pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -348,7 +348,8 @@ func PopulateExternalClusterInfo(context *clusterd.Context, namespace string, ow
 	}
 }
 
-func loadExternalClusterInfo(context *clusterd.Context, namespace string) (*cephclient.ClusterInfo, error) {
+// LoadExternalClusterInfo loads external cluster clusterInfo
+func LoadExternalClusterInfo(context *clusterd.Context, namespace string) (*cephclient.ClusterInfo, error) {
 	var err error
 	clusterInfo, _, _, err := LoadClusterInfo(context, namespace)
```

Comment:
We should merge LoadExternalClusterInfo and LoadClusterInfo both into the LoadClusterInfo method. Then everywhere just calls LoadClusterInfo() and it will do the right thing depending on the secrets it finds. In other words, I think we can just move these lines 359-381 inside the LoadClusterInfo method and get rid of LoadExternalClusterInfo.

Reply:
Yeah thought about that too!